src/runtime/virtcontainers/pkg/oci/utils.go

```diff
@@ -15,6 +15,7 @@ import (
 	"strconv"
 	"strings"
 	"syscall"
+	"encoding/json"
```

Comment:
```
src/runtime/virtcontainers/pkg/oci/utils.go:9: File is not `gofmt`-ed with `-s` (gofmt)
	"context"
```

Please, use gofmt to ensure the static checks are happy.

Reply:
@fidencio done (hopefully - I don't quite get what the problem is ;-) )